app/src/main/java/com/elavon/converge/core/TransactionManager.java

```diff
@@ -41,13 +42,13 @@
     private IPoyntSecurityService poyntSecurityService;
 
     protected Context context;
-    protected ConvergeClient convergeClient;
+    protected ConvergeService convergeService;
     protected Map<UUID, Transaction> transactionCache;
 
     @Inject
-    public TransactionManager(final Context context, final ConvergeClient convergeClient) {
+    public TransactionManager(final Context context, final ConvergeService convergeService) {
         this.context = context;
-        this.convergeClient = convergeClient;
+        this.convergeService = convergeService;
```

Comment:
`TransactionManager` uses `ConvergeService` instead of `ConvergeClient` directly